packages/data/src/index.js

```diff
@@ -79,6 +79,9 @@ export const select = defaultRegistry.select;
  * Given the name of a registered store, returns an object of the store's action creators.
  * Calling an action creator will cause it to be dispatched, updating the state value accordingly.
  *
+ * Note: If the action creator is a generator then a promise is returned.
```

Comment:
It's not necessarily limited to generators, is it? From the fact we're using just a simple promise middleware, I'd imagine any promise action could be supported?

Reply:
True, I clarified in 78185ab